mmtrack/core/track/interpolation.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+
+def tracklet_interpolation(results, min_frames=5, max_frames=20):
+    """Interpolate tracklet.
+
+    This function is proposed in
+    "ByteTrack: Multi-Object Tracking by Associating Every Detection Box."
+    `ByteTrack<https://arxiv.org/abs/2110.06864>`_.
+
+    Args:
+        results (ndarray): With shape (N, 7). Each row denotes
```

Comment:
"results" are not clear for an independent function in core, be more specific. 

We can discuss how to use "track" and "tracklet".